pageserver/src/tenant/timeline.rs

```diff
@@ -3464,7 +3464,7 @@ impl Timeline {
         let mut prev_lsn_end = first_level0_delta.layer_desc().lsn_range.end;
         let mut deltas_to_compact = vec![Arc::clone(first_level0_delta)];
         for l in level0_deltas_iter {
-            let lsn_range = l.layer_desc().lsn_range;
+            let lsn_range = &l.layer_desc().lsn_range;
```

Comment:
Well I fixupped this to wrong commit, should had been in previous.

Reply:
Moved in [ed68206](https://github.com/neondatabase/neon/pull/4936/commits/ed68206478aba991b653afb2250d753f0f21f6c6)